docs/container-overview.md

```diff
@@ -0,0 +1,173 @@
+
+Because the docker compose file is also used during development on one or more components most dependencies are actually accessed through the public IP of the host computer by default (marked with *"via docker host"*).
+This allows starting some containers from the docker compose and some in a local dev setup.
+However it can also lead to problems if the public IP has changed or a firewall blocks access to the public IP from the containers.
+
+If everything is started via docker-compose on Windows or Mac then it may be possible to use `host.docker.internal` as the public IP/domain.
+(see also <https://docs.docker.com/desktop/windows/networking/#use-cases-and-workarounds> or <https://docs.docker.com/desktop/mac/networking/#use-cases-and-workarounds>)
+
+On windows `host.docker.internal` is also known for the windows host.
+For mac this is not the case.
+A workaround for mac is to add the following entry to the `/etc/hosts` file (needs administrator rights!):
+
+```
+# manually added
+127.0.0.1   host.docker.internal
+```
+
+
+## container 
+
+Image: [`opentosca/container:latest`](https://hub.docker.com/r/opentosca/container) [GitHub](https://github.com/OpenTOSCA/container)
+
+Main coordinator of the whole OpenTOSCA machinery.
+Provides the application bus (for inter application communication) and the management bus (for management/deployment commands).
+Hosts a plan generator to generate BPEL deployment plans from topologies in a CSAR.
+Invokes management and deplyoment plans, relays implementation artifact (IA) commands to [engine-ia](#engine-ia).
+Manages metadata, logging, etc. of deployed instances.
+
+Ports:
+
+ *  1337: Main API
+ *  1883: MQTT (used for distributed/federated deployment of multiple container instances)
+ *  8081-8087: Application Bus and Management Bus
+
+### Dependencies
+
+ *  [engine-ia](#engine-ia) via docker-compose network\
+    Used to deploy and execute implementation artifacts when requested via the management bus.
+ *  [engine-plan](#engine-plan) via docker-compose network\
+    OR [engine-plan-bpmn](#engine-plan) via docker-compose network\
+    Used to run deployment and management plans. Only one engine is technically neccessary, but it must be able to execute all plans (workflows) in the CSAR archives in question.
+ *  [container-repository](#container-repository) via docker host\
+    Used to fill in missing information in CSARs (research feature).
+ *  [dind](#dind) not configured in compose file...\
+    Used as docker endpoint to create docker containers.
+
+
+## container-repository
+
+Image: [`opentosca/winery:latest`](https://hub.docker.com/r/opentosca/winery) [GitHub](https://github.com/OpenTOSCA/winery)
+
+Implementations of reasearch features.
+Used to fill in missing information of CSARs.
+
+Ports:
+
+ *  8091 (internal 8080): Main API
+
+### Dependencies
+
+ *  [container](#container) via docker host
+ *  [winery](#winery) via docker host\
```

Comment:
the container-repository is a winery, so I don't think it is a "dependency" in that sense

Reply:
Was genau konfigurieren die Env Variablen `WINERY_PORT` und `WINERY_HOSTNAME` bei `container-repository`? Ich hatte das zuerst als dependency interpretiert, macht aber keinen sinn so. Dann muss ich noch das Übersichtsbild anpassen.